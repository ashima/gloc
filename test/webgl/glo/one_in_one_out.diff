--- one_in_one_out.true	2011-08-15 20:10:31.000000000 -0700
+++ one_in_one_out.out	2011-10-28 14:07:40.000000000 -0700
@@ -10,8 +10,8 @@
   },
   "units": [
     {
-      "insym": [ "incoming" ],
-      "outsym": [ "main" ],
+      "insym": [],
+      "outsym": [],
       "inmac": [],
       "opmac": [],
       "outmac": [],
