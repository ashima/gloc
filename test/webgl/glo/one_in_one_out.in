void main() {
     gl_Position = vec3(incoming);
}
