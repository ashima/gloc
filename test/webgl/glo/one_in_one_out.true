{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "copyright": [ "", 2011, "" ],
    "author": {},
    "license": null,
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [ "incoming" ],
      "outsym": [ "main" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nvoid main() {\n     gl_Position = vec3(incoming);\n}"
    }
  ],
  "linkmap": { "0": "<stdin>#0" }
}
