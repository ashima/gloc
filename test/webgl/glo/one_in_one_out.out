{
  "glo": [ 1, 0, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": { "copyright": [ 2012, [ "", "" ] ] },
  "units": [
    {
      "insym": [ "incoming" ],
      "outsym": [ "main" ],
      "source":
        "#line 1 GLOC_0\nvoid main() {\n     gl_Position = vec3(incoming);\n}"
    }
  ],
  "linkmap": { "0": "#n=0" }
}
